### PR TITLE
Content doc for "Data and Analytics" developer section.

### PR DESCRIPTION
**Pull request by cj-technical**
Content doc for "Data and Analytics" developer section. 

## Description

Updated content topics and details surrounding "Data and Analytics".

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/1564

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
@cj-technical This looks great! Thanks for submitting this content addition. 

Only thing left to get this ready for prod is updating the `src/data/developer-docs-links.yaml` file (line 80 needs to be changed to `to:` instead of `path:`, ie: `to: /developers/docs/data-and-analytics/` for it to be properly linked to throughout the developer documents flow.

@cj-technical If you want to update this before it's merged feel free, otherwise I can fix this in a separate PR after merging.

**Comment by cj-technical** [comment]
@wackerow Happy to help. I updated the PR with a commit for the yaml update. Let me know if that squares things away.

**Review by wackerow — APPROVED** [review]
@cj-technical LGTM!

File: src/content/developers/docs/data-and-analytics/index.md [hunk 1]
```diff
@@ -3,9 +3,37 @@ title: Data and analytics
...
 
+The [Graph Network](https://thegraph.com/) is a decentralized indexing protocol for organizing blockchain data. Instead of building and managing off-chain and centralized data stores to aggregate on-chain data, with The Graph, developers can build serverless applications that run entirely on public infrastructure.
+
+Using [GraphQL](https://graphql.org/), developers can query any of the curated open APIs, known as sub-graphs, to acquire the necessary information they need to drive their dApp. By querying these indexed sub-graphs, Reports and dApps not only get performance and scalability benefits but also the built in accuracy provided by network consensus. As new improvements and/or sub-graphs are added to the network, your projects can rapidly iterate to take advantage of these enhancements.
+
 ## Block explorers {#block-explorers}
+
+Many [Block Explorers](/en/developers/docs/block-explorers#block-explorers) offer [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) [API](https://en.wikipedia.org/wiki/API) gateways that will provide developers visibility into real-time data on blocks, transactions, miners, accounts, and other on-chain activity.
```
Comment (ryancreatescopy) [inline src/content/developers/docs/data-and-analytics/index.md:29]: don't think we need language path hardcoded here or anywhere in this doc

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]

File: src/content/developers/docs/data-and-analytics/index.md (hunk 1 above)
Comment (cj-technical) [reply src/content/developers/docs/data-and-analytics/index.md:29]: Just the wiki links or the content link, as well?

Comment (cj-technical) [reply src/content/developers/docs/data-and-analytics/index.md]: @ryancreatescopy  Language references in the wiki links are removed and checked in.

Comment (ryancreatescopy) [reply src/content/developers/docs/data-and-analytics/index.md]: apologies I was referring to the ethereum.org links to `(/developers/docs/block-explorers#block-explorers)`

Comment (cj-technical) [reply src/content/developers/docs/data-and-analytics/index.md]: No worries. Updated those as well.

**Review by wackerow — APPROVED** [review]

**Comment by ryancreatescopy** [comment]
@all-contributors please add @cj-technical for content.

**Pull request merged**